.gitignore

```diff
@@ -14,3 +14,4 @@
 mkmf.log
 *.gem
 .byebug_history
+.idea/*
```

Comment:
Should put this into a global gitignore. See https://gist.github.com/subfuzion/db7f57fff2fb6998a16c

Reply:
Good idea, will clean up